tutorials/Intro-to-geospatial-analysis/index.md

```diff
@@ -0,0 +1,448 @@
+---
+title: An Introduction to Geospatial Analysis Using Google Earth Engine
```

Comment:
You can omit "Using Google Earth Engine", since this title will show under the left menu in the EE docs.

Reply:
Done.